charts/incubator/adguardhome-sync/questions.yaml

```diff
@@ -9,44 +9,154 @@ questions:
 # Include{replica1}
 # Include{controllerExpertExtraArgs}
   - variable: env
-    group: "App Configuration"
-    label: "Image Environment"
+    group: App Configuration
+    label: Image Environment
     schema:
       additional_attrs: true
       type: dict
       attrs:
-        - variable: CONFIGFILE
-          label: "CONFIGFILE"
-          description: "Set a custom config file."
+        - variable: original
+          label: Original Server Config
           schema:
-            type: string
-            default: ""
-        - variable: UMASK
-          label: "UMASK"
-          description: "Container Variable UMASK"
+            type: boolean
+            default: true
+            show_subquestions_if: true
+            subquestions:
+              - variable: ORIGIN_URL
+                label: Origin URL
+                schema:
+                  type: string
+                  default: ""
+                  required: true
+              - variable: ORIGIN_USERNAME
+                label: Origin Username
+                schema:
+                  type: string
+                  default: ""
+                  required: true
+              - variable: ORIGIN_PASSWORD
+                label: Origin Password
+                schema:
+                  type: string
+                  default: ""
+                  required: true
+        - variable: replica
+          label: Replica Server Config
+          schema:
+            type: boolean
+            default: true
+            show_subquestions_if: true
+            subquestions:
+              - variable: REPLICA_URL
+                label: Replica URL
+                schema:
+                  type: string
+                  default: ""
+                  required: true
+              - variable: REPLICA_USERNAME
+                label: REPLICA Username
+                schema:
+                  type: string
+                  default: ""
+                  required: true
+              - variable: REPLICA_PASSWORD
+                label: REPLICA Password
+                schema:
+                  type: string
+                  default: ""
+                  required: true
+        - variable: CRON
+          label: Cron
           schema:
             type: string
-            default: ""
+            default: "*/10 * * * *"
+        - variable: RUNONSTART
+          label: Run On Start
+          schema:
+            type: boolean
+            default: true
+        - variable: sync_features
```

Comment:
Please always have the env's and other app config options in the values.yaml.
Even if they have to be commented out